OptimizelySDK/Logger/DefaultLogger.cs

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-using System.Diagnostics;
+using System;
```

Comment:
where is `System` used?

Reply:
"Console" class is defined in the "System" namespace.

Reply:
Line 38 of DefaultLogger.cs is "Console.WriteLine(line);"

Reply:
Are we linking/importing all system files here like in Java? Would System.Console be better?